clang/docs/ReleaseNotes.rst

```diff
@@ -933,6 +933,14 @@ X86 Support
 - Support ISA of ``AVX10.1``.
 - ``-march=pantherlake`` and ``-march=clearwaterforest`` are now supported.
 - Added ABI handling for ``__float128`` to match with GCC.
+- Emit warnings for options to enable knl/knm specific ISAs: AVX512PF, AVX512ER
+  and PREFETCHWT1. From next version (LLVM 19), these ISAs' intrinsic supports
+  will be deprecated:
+  * intrinsic series of *_exp2a23_*
+  * intrinsic series of *_rsqrt28_*
+  * intrinsic series of *_rcp28_*
+  * intrinsic series of *_prefetch_i[3|6][2|4]gather_*
+  * intrinsic series of *_prefetch_i[3|6][2|4]scatter_*
```

Comment:
And `_m_prefetchw`

Reply:
That's not knl specific is it? Wasn't that part of 3dnow? The knl one is for prefetchwt1.

Reply:
Thanks @topperc ! You are correct, I forgot that.
KNL has a seperate feature called `prefetchwt1`, but the intrinsic/mnemonic is the same.